app/assets/javascripts/test/sagas/prefetch_saga.spec.js

```diff
@@ -7,6 +7,11 @@ import Model from "oxalis/model";
 import { expectValueDeepEqual, execCall } from "../helpers/sagaHelpers";
 import DATASET from "../fixtures/dataset_server_object";
 
+// temporary workaround for DataLayer class vs DataLayer object discrepancy
```

Comment:
I'll resolve this in another PR when the dependency of the DataLayer instances is reduced/removed.